superset/daos/tag.py

```diff
@@ -122,7 +122,7 @@ def get_by_name(name: str, type_: TagType = TagType.custom) -> Tag:
             .first()
         )
         if not tag:
-            tag = get_tag(name, db.session, type_)
+            tag = get_or_create_tag(name, db.session, type_)
         return tag
```

Comment:
While you're at it, do you think it makes sense to also re-name this to `get_or_create_by_name`? The docstring is also a bit confusing currently as it says first "none otherwise" but then there's the "important!" block that says one would be created.